tests/metrics/test_utilities.py

```diff
@@ -46,3 +78,66 @@ def test_query_string(a, b):
     assert b == 'b and a and b'
     assert b == QueryString('b and a and b')
     assert isinstance(b, QueryString)
+
+
+def test_get_susceptible_person_time(ages_and_bins, sexes, observer_config):
+    ages, age_bins = ages_and_bins
+    disease = 'test_disease'
+    states = [f'susceptible_to_{disease}', disease]
+    pop = pd.DataFrame(list(product(ages, sexes, states)), columns=['age', 'sex', disease])
+    pop['alive'] = 'alive'
+    # Shuffle the rows
+    pop = pop.sample(frac=1).reset_index(drop=True)
+
+    year = 2017
+    step_size = pd.Timedelta(days=7)
+
+    person_time = get_susceptible_person_time(pop, observer_config, disease, year, step_size, age_bins)
+
+    values = set(person_time.values())
+    assert len(values) == 1
+    expected_value = to_years(step_size)*len(pop)/2
+    if observer_config['by_sex']:
+        expected_value /= 2
+    if observer_config['by_age']:
+        expected_value /= len(age_bins)
+    assert np.isclose(values.pop(), expected_value)
+
+    # Doubling pop should double person time
+    pop = pd.concat([pop, pop], axis=0, ignore_index=True)
+
+    person_time = get_susceptible_person_time(pop, observer_config, disease, year, step_size, age_bins)
+
+    values = set(person_time.values())
+    assert len(values) == 1
+    assert np.isclose(values.pop(), 2*expected_value)
+
+
+def test_get_disease_event_counts(ages_and_bins, sexes, observer_config):
+    ages, age_bins = ages_and_bins
+    disease = 'test_disease'
+    event_time = pd.Timestamp('1-1-2017')
+    states = [event_time, pd.NaT]
+    pop = pd.DataFrame(list(product(ages, sexes, states)), columns=['age', 'sex', f'{disease}_event_time'])
+    # Shuffle the rows
+    pop = pop.sample(frac=1).reset_index(drop=True)
+
+    counts = get_disease_event_counts(pop, observer_config, disease, event_time, age_bins)
+
+    values = set(counts.values())
+    assert len(values) == 1
+    expected_value = len(pop) / 2
+    if observer_config['by_sex']:
+        expected_value /= 2
+    if observer_config['by_age']:
+        expected_value /= len(age_bins)
+
+    # Doubling pop should double person time
+    pop = pd.concat([pop, pop], axis=0, ignore_index=True)
+
+    person_time = get_disease_event_counts(pop, observer_config, disease, event_time, age_bins)
```

Comment:
this should probably be counts instead of person_time?

Reply:
Will fix in the next PR